code/modules/ninja/suit/n_suit_verbs/energy_net_nets.dm

```diff
@@ -53,31 +59,32 @@ It is possible to destroy the net by the occupant or someone else.
 	if(ishuman(affecting))
 		var/mob/living/carbon/human/H = affecting
 		for(var/obj/item/W in H)
-			if(W == H.w_uniform)
-				continue//So all they're left with are shoes and uniform.
-			if(W == H.shoes)
-				continue
-			H.dropItemToGround(W)
+			if(!is_type_in_typecache(W, weeb_jail_essential_items) && H.temporarilyRemoveItemFromInventory(W)) //Remove all non-essential items, including implants.
+				if(istype(W, /obj/item/restraints/)) //No need for restraints once they're inside the weeb jail.
+					W.forceMove(get_turf(H.loc))
+					continue
+				else
+					W.forceMove(get_turf(H.loc))
```

Comment:
What's the point of this else?

Reply:
It's a leftover from gulag TP code, I'll remove it when I'm back from work.